### PR TITLE
Adds the "why" for `next/image` optimization

### LINKED ISSUE #30220: Add more data to docs on why image optimization is important
### Describe the feature you'd like to request

The top of the Image Optimization page now has a list of next/image performance benefits, but it also might be helpful for folks to hear more specific data about the problem that it's trying to solve (i.e.why images are such a big contributor to perf issues on the web).

### Describe the solution you'd like

Add stats to the Image Optimization intro like:

- Unoptimized images account for over 70% of the total page weight in bytes at the 90th percentile.  (https://almanac.httparchive.org/en/2020/page-weight#image-bytes) 
- The number of images on a page is the second greatest predictor of conversions of users visiting websites. (https://almanac.httparchive.org/en/2019/page-weight#bigger-complex-pages-can-be-bad-for-your-business)

### Describe alternatives you've considered

There are a bunch more stats in the Web Almanac that could be interesting to cite.

### PR DESCRIPTION
**Pull request by manovotny**
Give a touch more context as to why one would want to use image optimization.

Fixes #30220

### CODE REVIEW

File: docs/02-app/01-building-your-application/05-optimizing/01-images.mdx [hunk 1]
```diff
@@ -18,7 +18,9 @@ related:
 
 </details>
 
-The Next.js Image component extends the HTML `<img>` element with:
+According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/media#images) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
+
+The Next.js Image component extends the HTML `<img>` element with features that make image optimization easier to achieve:
```
Comment (styfle) [inline docs/02-app/01-building-your-application/05-optimizing/01-images.mdx:23]: "make image optimization easier to achieve" sounds like the component doesn't work in all cases or that it is not zero config.

Can we word this differently or drop it?

Comment (manovotny) [reply docs/02-app/01-building-your-application/05-optimizing/01-images.mdx]: > sounds like the component doesn't work in all cases or that it is not zero config.

Not at all how I intended it. Took a stab at another phrasing.

File: docs/02-app/01-building-your-application/05-optimizing/01-images.mdx [hunk 2]
```diff
@@ -18,7 +18,9 @@ related:
 
 </details>
 
-The Next.js Image component extends the HTML `<img>` element with:
+According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/media#images) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
+
+The Next.js Image component extends the HTML `<img>` element with features for image optimization:
```
Comment (styfle) [inline docs/02-app/01-building-your-application/05-optimizing/01-images.mdx:23]:
```suggestion
The Next.js Image component extends the HTML `<img>` element with features for automatic image optimization:
```

File: docs/02-app/01-building-your-application/05-optimizing/01-images.mdx [hunk 3]
```diff
@@ -18,7 +18,9 @@ related:
 
 </details>
 
-The Next.js Image component extends the HTML `<img>` element with:
+According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/media#images) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
```
Comment (styfle) [inline docs/02-app/01-building-your-application/05-optimizing/01-images.mdx:21]:
```suggestion
According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/page-weight#content-type-and-file-formats) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
```

This link seems wrong. Seems like it should link to the page weight graph instead?

**Review by styfle — APPROVED** [review]

**Review by hydaryali46 — COMMENTED** [review]
Institution concerned European Research

**Pull request merged**